config/config.go

```diff
@@ -3,11 +3,10 @@ package config
 import (
 	"bytes"
 	"crypto/tls"
-	_ "embed"
 	"errors"
 	"fmt"
 	"io"
-	"log"
+	"log" //nolint:revive
```

Comment:
Shouldn't this be excluded by [that line in .golangci.yml](https://github.com/influxdata/telegraf/blob/b5c618ee8b39dc3bd7a199994fa0bb815523a487/.golangci.yml#L127)?

Reply:
Good catch, maybe this is a bug with vscode because with I run `golangci-lint run config/` the linter error doesn't show up, but vscode is complaining about it.